### ISSUE TITLE
Rocket Chat app stuck on Splash screen and  TypeError: Cannot read property 'default' of undefined, js engine: hermes

### ISSUE THREAD

**Issue by rahulcomlink** [RocketChat/Rocket.Chat.ReactNative#5053]
### Environment Information:

- Rocket.Chat App Version: 4.37.1 
- Device Name:  Vivo1920
- OS Version:  Android 12

### Query:

I need a information regarding which android build variant should be use for white labelled app. I need notification too.

### Getting Error

TypeError: Cannot read property 'default' of undefined, js engine: hermes

<img width="436" alt="Screenshot 2023-05-02 at 4 10 17 PM" src="https://user-images.githubusercontent.com/95681028/235645229-a5120c50-0e43-4148-b1b5-c16414328699.png">

### Step to Reproduce 

I am unable to run app from both CLE & android studio , app screen is stuck on splash screen.

**Comment by diegolmello** [comment]
> TypeError: Cannot read property 'default' of undefined, js engine: hermes

This error is on javascript side and you should be able to debug it with Flipper or check logs.

**Comment by yogesh6260** [comment]
Yes the same issue happening in my 'vivo 1820' Android phone, when I run the app on building source code. It will show this blue screen and stuck nothing happens after the blue screen. I had built the latest source code, using 'yarn android' and the build was SUCCESSFUL. But don't know why my app splash screen is stuck. Please, help me with the solution.

**Comment by yogesh6260** [comment]
> 
I have the same issue, I solved by taking help from  - 
[https://stackoverflow.com/questions/55763428/react-native-error-enospc-system-limit-for-number-of-file-watchers-reached](url)

**Comment by thakkarnetram** [comment]
I was facing the same issue , i installed cmake  and built the app again and reloaded . It worked for me !
